script.js

```diff
@@ -0,0 +1,27 @@
+//Utworzenie funkcji konstruującej Button
```

Comment:
comments in code only in english :)

Reply:
To be honest with you, these comments are for me :) I wanted to know what is happening in this code step by step and ... Polish is still my native language :)

Reply:
I will remove them.